thinkhazard/templates/report.jinja2

```diff
@@ -108,34 +108,34 @@
             {% endif %}
           </div>
           {% if hazard %}
-          <div class="text-center">
+          <div id="level" class="text-center">
             {{ hazard.hazardtype.mnemonic}} hazard Level
           </div>
-          <div class="text-center">
+          <div id="location" class="text-center">
             {{ division.name }}
           </div>
           <div id="legend">
             <ul>
-              <li class="level-HIG">
+              <li class="level-HIG col-md-6">
                 <i class="square"></i>
                 High
               </li>
-              <li class="level-MED">
+              <li class="level-MED col-md-6">
                 <i class="square"></i>
                 Medium
               </li>
-              <li class="level-LOW">
+              <li class="level-LOW col-md-6">
                 <i class="square"></i>
                 Low
               </li>
-              <li class="level-NPR">
+              <li class="level-NPR col-md-6">
```

Comment:
Alignment of legend items is not perfect. We'll probably need to address this later.